testsuite/features/support/twopence_init.rb

```diff
@@ -277,8 +277,8 @@ def file_inject(node, local_file, remote_file)
                   'debian10_ssh_minion'       => $debian10_ssh_minion,
                   'sle11sp4_buildhost'        => $sle11sp4_buildhost,
                   'sle11sp3_terminal'         => $sle11sp3_terminal,
-                  'sle12sp4_buildhost'        => $sle12sp4_buildhost,
-                  'sle12sp4_terminal'         => $sle12sp4_terminal,
+                  'sle12sp5_buildhost'        => $sle12sp5_buildhost,
+                  'sle12sp5_terminal'         => $sle12sp5_terminal,
```

Comment:
I hope you checked there is no need to synchronize a different product/channels :)

Reply:
Like here?
```

@scc_credentials
  Scenario: Enable sles12-sp4-pool-x86_64
    When I execute mgr-sync "add channel sles12-sp4-pool-x86_64"
    And I execute mgr-sync "list channels"
    Then I should get "[I] SLES12-SP4-Pool for x86_64 SUSE Linux Enterprise Server 12 SP4 x86_64 [sles12-sp4-pool-x86_64]"
    And I should get "    [I] SLES12-SP4-Updates for x86_64 SUSE Linux Enterprise Server 12 SP4 x86_64 [sles12-sp4-updates-x86_64]"
    And I should get "    [ ] SLE-Module-Containers12-Pool for x86_64 Containers Module 12 x86_64 [sle-module-containers12-pool-x86_64-sp4]"

@scc_credentials
  Scenario: Enable sle-module-containers12-pool-x86_64-sp4
    When I execute mgr-sync "add channel sle-module-containers12-pool-x86_64-sp4"
    And I execute mgr-sync "list channels"
    Then I should get "[I] SLES12-SP4-Pool for x86_64 SUSE Linux Enterprise Server 12 SP4 x86_64 [sles12-sp4-pool-x86_64]"
    And I should get "    [I] SLE-Module-Containers12-Pool for x86_64 Containers Module 12 x86_64 [sle-module-containers12-pool-x86_64-sp4]"
    And I should get "    [I] SLE-Module-Containers12-Updates for x86_64 Containers Module 12 x86_64 [sle-module-containers12-updates-x86_64-sp4]"
```

Reply:
I said nothing, we are good, this was for CI, and we are in BV features here:

``` 

  Scenario: Add SUSE Linux Enterprise Server 12 SP4
    When I follow the left menu "Admin > Setup Wizard > Products"
    And I wait until I do not see "Loading" text
    And I enter "SUSE Linux Enterprise Server 12 SP4" as the filtered product description
    And I select "SUSE Linux Enterprise Server 12 SP4 x86_64" as a product
    Then I should see the "SUSE Linux Enterprise Server 12 SP4 x86_64" selected
    When I open the sub-list of the product "SUSE Linux Enterprise Server 12 SP4 x86_64"
    And I select "SUSE Linux Enterprise Server LTSS 12 SP4 x86_64" as a product
    Then I should see the "SUSE Linux Enterprise Server LTSS 12 SP4 x86_64" selected
    When I click the Add Product button
    And I wait until I see "SUSE Linux Enterprise Server 12 SP4 x86_64" product has been added

  Scenario: Add SUSE Linux Enterprise Server 12 SP5
    When I follow the left menu "Admin > Setup Wizard > Products"
    And I wait until I do not see "Loading" text
    And I enter "SUSE Linux Enterprise Server 12 SP5" as the filtered product description
    And I select "SUSE Linux Enterprise Server 12 SP5 x86_64" as a product
    Then I should see the "SUSE Linux Enterprise Server 12 SP5 x86_64" selected
    When I click the Add Product button
    And I wait until I see "SUSE Linux Enterprise Server 12 SP5 x86_64" product has been added
```

Reply:
I did check, but at that time it was all about sle 15 sp3 and not sle 12 sp5... then i discovered that sle 12 sp5 was broken too.

So yes, I'm likely to have missed that one. Thank you!

Reply:
Ah, you already noticed that. Good.